modules/programs/neovim.nix

```diff
@@ -16,6 +16,22 @@ let
     merge = mergeOneOption;
   };
 
+  cocPluginPaths = map (p:
+    let
+      # coc.nvim is not able to expand pack paths, so we do it manually
+      plugin = p.plugin or p;
+      name = plugin.pname or plugin.name;
+    in "/pack/home-manager/start/${name}") cfg.coc.plugins;
+
+  plugins = cfg.plugins ++ optionals cfg.coc.enable (cfg.coc.plugins
+    ++ singleton {
+      plugin = pkgs.vimPlugins.coc-nvim;
+      config = concatMapStringsSep "\n"
+        (p: ''let &runtimepath .= ',' . split(&packpath, ',')[0] . "${p}"'')
```

Comment:
Trying to use `packpath` here seems silly, I should just use the absolute paths I think. And maybe even omit them from the pack entirely?